fe/fe-core/src/main/java/com/starrocks/transaction/GlobalTransactionMgr.java

```diff
@@ -314,6 +314,10 @@ public long beginTransaction(long dbId, List<Long> tableIdList, String label, TU
                 checkValidTimeoutSecond(timeoutSecond, Config.max_stream_load_timeout_second,
                         Config.min_load_timeout_second);
                 break;
+            case LAKE_COMPACTION:
+                checkValidTimeoutSecond(timeoutSecond, Config.max_lake_compaction_timeout_second,
+                        Config.min_load_timeout_second);
```

Comment:
Why not just skip checking the timeout range for compaction transactions?

Reply:
👍 Yeah, that sounds like a better option. As for lake compaction transaction, we can just skip the timeout value check.